content/docs/alerting/routing-tree-editor.html

```diff
@@ -0,0 +1,71 @@
+---
+title: Routing tree editor
+sort_rank: 4
+nav_icon: sliders
+---
+<style>
```

Comment:
This whole page could use a custom nanoc layout (similar to the landing page etc.) that would include all the right headers etc. If we decide that it shouldn't be part of the docs itself, that would make even more sense...